momento-sdk/src/intTest/java/momento/sdk/SimpleCacheClientTest.java

```diff
@@ -190,4 +191,75 @@ public void initializesSdkAndCanHitControlPlaneForUnreachableDataPlane() {
           .hasMessageContaining("server was unable to handle the request");
     }
   }
+
+  @Test
+  public void shouldReturnCacheIncrementedValuesWithStringField() {
+    final String field = "field";
+
+    CacheIncrementResponse incrementResponse =
+        target.increment(cacheName, field, 1, DEFAULT_TTL_SECONDS).join();
+
+    assertThat(incrementResponse).isInstanceOf(CacheIncrementResponse.Success.class);
+    assertThat(((CacheIncrementResponse.Success) incrementResponse).valueNumber()).isEqualTo(1);
+
+    incrementResponse = target.increment(cacheName, field, 50, DEFAULT_TTL_SECONDS).join();
+
+    assertThat(incrementResponse).isInstanceOf(CacheIncrementResponse.Success.class);
+    assertThat(((CacheIncrementResponse.Success) incrementResponse).valueNumber()).isEqualTo(51);
+
+    incrementResponse = target.increment(cacheName, field, -1051, DEFAULT_TTL_SECONDS).join();
+
+    assertThat(incrementResponse).isInstanceOf(CacheIncrementResponse.Success.class);
+    assertThat(((CacheIncrementResponse.Success) incrementResponse).valueNumber()).isEqualTo(-1000);
+
+    CacheGetResponse getResp = target.get(cacheName, field).join();
+    assertThat(getResp).isInstanceOf(CacheGetResponse.Hit.class);
+    assertThat(((CacheGetResponse.Hit) getResp).valueString()).isEqualTo("-1000");
+  }
+
+  @Test
+  public void shouldReturnCacheIncrementedValuesWithUint8ArrayField() {
+    final byte[] field = "field".getBytes();
+
+    CacheIncrementResponse incrementResponse =
+        target.increment(cacheName, field, 1, DEFAULT_TTL_SECONDS).join();
+
+    assertThat(incrementResponse).isInstanceOf(CacheIncrementResponse.Success.class);
+    assertThat(((CacheIncrementResponse.Success) incrementResponse).valueNumber()).isEqualTo(1);
+
+    incrementResponse = target.increment(cacheName, field, 50, DEFAULT_TTL_SECONDS).join();
+
+    assertThat(incrementResponse).isInstanceOf(CacheIncrementResponse.Success.class);
+    assertThat(((CacheIncrementResponse.Success) incrementResponse).valueNumber()).isEqualTo(51);
+
+    incrementResponse = target.increment(cacheName, field, -1051, DEFAULT_TTL_SECONDS).join();
+
+    assertThat(incrementResponse).isInstanceOf(CacheIncrementResponse.Success.class);
+    assertThat(((CacheIncrementResponse.Success) incrementResponse).valueNumber()).isEqualTo(-1000);
+
+    CacheGetResponse getResp = target.get(cacheName, field).join();
+    assertThat(getResp).isInstanceOf(CacheGetResponse.Hit.class);
+    assertThat(((CacheGetResponse.Hit) getResp).valueString()).isEqualTo("-1000");
+  }
+
+  @Test
+  public void shouldFailCacheIncrementedValuesWhenNullCacheName() {
+    final String field = "field";
+
+    try {
+      target.increment(null, field, 1, DEFAULT_TTL_SECONDS).join();
```

Comment:
I don't think this test should pass since the method doesn't throw anymore. The assertion is skipped because no exception is thrown. For a method that is supposed to throw you would need to assert that an exception was thrown using `assertThatExceptionOfType(InvalidArgumentException.class).isThrownBy(...)` or similar. In this case you just need to assert that the response type is an error with the expected exception as a cause.